pkg/cloud/openstack/assets/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: openstack-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    app: openstack-cloud-controller-manager
+spec:
+  selector:
+    matchLabels:
+      app: openstack-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: openstack-cloud-controller-manager
+    spec:
+      hostNetwork: true
+      serviceAccount: cluster-cloud-controller-manager
```

Comment:
This SA apparently does not exist. @Fedosin

Reply:
Also, it should probably follow or inherit or extend this SA with roles: https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/15/files#diff-2b0c81c2d79e6551f107545166e6281fbb47a343b7080d29c87c0c2637cd76b4R1-R134 

As far as I know openstack CCM definitely needs access to manage nodes: https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/15/files#diff-2b0c81c2d79e6551f107545166e6281fbb47a343b7080d29c87c0c2637cd76b4R30-R39

Reply:
```suggestion
      serviceAccount: cloud-controller-manager
```